### PR TITLE
After Saving a series edit reset the batch editor

### PR DESCRIPTION
**Pull request by bp323**
Current the batch editor is accessible after an Edit -> Save session, with the input fields not cleared.

When user clicks on Save, all the input fields should be cleared / reset, and the batch editor should go into inactive state

timetable_administration

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/gh/js/views/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -337,6 +337,22 @@ define(['gh.core', 'gh.constants', 'moment', 'gh.calendar', 'gh.admin-event-type
...
+     *
+     * @private
+     */
+    var resetBatchHeader = function() {
+        // Reset the title
+        $('#gh-batch-edit-title').val('');
+        // Reset the location
+        $('#gh-batch-edit-location').val('');
+        // Reset the type
+        $('#gh-batch-edit-type').val('Lecture');
```
Comment (Coenego) [inline shared/gh/js/views/gh.admin-batch-edit.js]: Can we use the default value here from the constants?

**Comment by Coenego** [comment]
Reassigning to @bp323 for follow-up

**Comment by bp323** [comment]
Back to @Coenego for review

**Pull request merged**